Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -111,6 +111,12 @@ inline std::shared_ptr<core::Filter> Filter(absl::string_view key,
   return Filter(key, op, model::FieldValue::IntegerValue(value));
 }
 
+inline std::shared_ptr<core::Filter> Filter(absl::string_view key,
+                                            absl::string_view op,
+                                            float value) {
```

Comment:
Nit: why not accept a `double` instead of a `float`? The literal for doubles is slightly less verbose, it works better with the call to `DoubleValue` below (unless it takes `float`s for some reason?), and the test above uses a `double` literal anyway.

Reply:
Fixed. (Not sure why I did that.)